src/api/forms/form-metadata-repository.js

```diff
@@ -35,9 +35,9 @@ export async function listForms() {
  * @param {string} formId - ID of the form
  * @returns {Promise<FormConfiguration>} - form configuration
  */
-const getFormMetadata = async (formId) => {
+export const getFormMetadata = async (formId) => {
```

Comment:
Sorry, missed this one

If we can keep function names for stack traces etc

```suggestion
export async function getFormMetadata(formId) {
```

<br>

Probably noticed me adding [`--enable-source-maps`](https://nodejs.org/api/cli.html#--enable-source-maps) (see https://github.com/DEFRA/forms-manager/commit/76912175c4e84a7d3616ea1747864aa87240ce18) so we get output like:

```console
    err: {
      "type": "TypeError",
      "message": "Cannot read properties of undefined (reading 'collection')",
      "stack":
          TypeError: Cannot read properties of undefined (reading 'collection')
              at populateApi (/path/to/project/forms-manager/src/helpers/db/populate-api.js:12:33)
              at Object.register (/path/to/project/forms-manager/src/helpers/db/populate-db.js:16:15)
              at internals.Server.register (/path/to/project/forms-manager/node_modules/@hapi/hapi/lib/server.js:494:35)
              at createServer (/path/to/project/forms-manager/src/api/server.js:61:16)
              at startServer (/path/to/project/forms-manager/src/index.js:17:18)
```